### PR TITLE
Auth

### PR DESCRIPTION
**Pull request by idanstark42**

### CODE REVIEW

File: fllscoring-configs/multiusers.conf.js [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+    scripts: "./src/**/*.js",
+    assets: ["./src/**/*",
+           "./challenges/**/*"],
+    dirs:  ["./src",
+          "./src/data",
+          "./challenges"],
+    users: [{
+        username: 'admin',
+        password: 'admin_password',
+        pages: ['scoresheet','teams','scores','ranking','settings'],
```
[the remark this answers is not in the document]
Comment (idanstark42) [reply fllscoring-configs/multiusers.conf.js]: Done

**Comment by idanstark42** [comment]
You're right. This repository is definitely not the right place for configurations. This is something that will require it's own service, or to be integrated into the launcher. This is really a matter of time vs. effort

**Comment by ThinkRedstone** [comment]
@idanstark42 Can we maybe move it to the launcher? You didn't actually change any files, so it's just a matter of copy-pasting a few files, and then changing the arguments in modules.yml and calling generate_passwords from the main launcher process. It makes a lot of more sense to me- someone who only wants to run the display and the scoring apps and thus needs to use relays from this package doesn't need to also have the config files for running the whole system.

**Comment by idanstark42** [comment]
I don't want to move this alone, without the relays configurations. This repo currently contains all of the configurations for the different apps, in all sorts of usage protocols, such as only display-scoring, display-scoring with authentication, scoring-clock with authentication, and so forth. So when we move this, it should be to another repository for configuration, because of you want that you don't need the launcher. So when we make that move, I want to do it quickly like a bandaid, but it's not for now

**Pull request merged**